qiita_db/study.py

```diff
@@ -143,6 +145,11 @@ class Study(QiitaStatusObject):
     _table = "study"
     # The following columns are considered not part of the study info
     _non_info = {"email", "study_status_id", "study_title"}
+    # The following tables are considered part of info
+    _info_cols = frozenset(chain(
```

Comment:
None of this information will change at runtime, right? If so, I think this could be cached in `qiita_pet.__init__` and done once